docs/installation_configuration_guide/Common.md

```diff
@@ -62,14 +64,17 @@ See [development configuration](https://github.com/TouK/nussknacker/blob/staging
 
 ## Environment variables
 
-Environment variables are described in [Installation guide](../installation/Installation.md), they are mostly helpful in the docker setup.
+Environment variables are described in the [Installation guide](../installation/Installation.md), they are mostly helpful in the docker setup.
 
 ## Conventions
 
 * We use HoCon (see [introduction](https://github.com/lightbend/config#using-hocon-the-json-superset) or [full specification](https://github.com/lightbend/config/blob/master/HOCON.md) for details) as our main configuration format. [Lightbend config library](https://github.com/lightbend/config/tree/master) is used for parsing configuration files - you can check [documentation](https://github.com/lightbend/config#standard-behavior) for details on conventions of file names and merging of configuration files.
-* `nussknacker.config.locations` Java system property (`CONFIG_FILE` environment variable for Docker image) defines location of configuration files (separated by comma). The files are read in order, entries from later files can override the former (using HoCon fallback mechanism) - see docker demo for example:
+* `nussknacker.config.locations` Java system property (`CONFIG_FILE` environment variable for Docker image) defines location of configuration files (separated by comma). The files are read in order, entries from later files can override the former (using HoCon fallback mechanism). This mechanism is used to extend or override default configuration contained in the [minimal configuration file](#minimal-configuration-file)  - see docker demo for example:
   * [setting multiple configuration files](https://github.com/TouK/nussknacker-quickstart/blob/main/docker/common/docker-compose.yml#L13)
   * [file with configuration override](https://github.com/TouK/nussknacker-quickstart/blob/main/docker/streaming/nussknacker/nussknacker.conf)
 * If `config.override_with_env_vars` Java system property is set to true, it is possible to override settings with env variables. This property is set to true in the official Nussknacker Docker image.
 
 It’s important to remember that model configuration is prepared a bit differently. Please read [model configuration](./model/ModelConfiguration.md) for the details. 
+
+## What is next?
+Most likely you will want to configure enrichers - they are configured under the modelConfig configuration key - see the extract of the configuration file above. The details of enrichers configuration are in the [Integration chapter](../integration/) of the documentation. 
```

Comment:
1. modelConfig -> `modelConfig`
2. "see the extract of the configuration file above." -> IMO it'd be better to have a section that we could refer to